build/build-treemap.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
```

Comment:
is this the long-term plan for this app? maintaining *another* bespoke bundler wasn't high on my list of things we should aspire to :/

I'm assuming this might also grow in complexity as we layer in the rest of the features, but maybe you have a different expectation?

Reply:
I should have mentioned this. This is exactly the same file as `build-viewer`, but different inputs.

> I'm assuming this might also grow in complexity as we layer in the rest of the features, but maybe you have a different expectation?

This is the final state, save for an addition to the js inputs.

Reply:
I'm ok with that, but I want one more person to weigh in. I can see a case for keeping it separate (the normal arguments against DRY).

Reply:
That's fair. I would agree with you if it weren't code that I think is a mistake we're maintaining in the first place, let alone a *new* copy 😆

Reply:
https://github.com/GoogleChrome/lighthouse/pull/11564